tasks/pipeline-dbconf.yml

```diff
@@ -36,6 +36,7 @@
     encoding: "utf8"
     collation: "utf8_unicode_ci"
   register: "createdb_result"
+  when: 'archivematica_src_am_database_host == "localhost"'
```

Comment:
I think we should move MySQL database user and db creation outside of archivematica role, and do it in the percona role step.

From my point of view, we should treat elasticearch, mysql and gearman as external dependencies needed for archivematica to run, and manage them using their roles, not the archivematica one. For mysql, we only need to update the vars used to reflect something like this:

'
mysql_databases: 
   - name: MCP
     collation: utf8

mysql_users:
   - name: archivematica
     pass: {{ secret_am_pass }}
     #host: localhost 

archivematica_src_am_database_name: {{ mysql_databases[0].name }}

'
(This won't work as is, but you get the idea...)